README.md

```diff
@@ -81,3 +81,212 @@ The following options are specific to this plugin:
   include A/B testing or localization specific output. Note: if you
   invoke the callback more than once, you should change the output
   filename to avoid overwriting previous invocations' output.
+
+
+## Examples
```

Comment:
@eoneill Okay, I’ve moved the example write-ups into their file, as requested. I kept a shorter version of the first (minimal) example in the README and inserted link that should take the reader to the rendered EXAMPLES file once this PR is merged.